src/components/dropdown/dropdown.tsx

```diff
@@ -464,32 +465,34 @@ export class Dropdown implements InteractiveComponent {
 
   transitionEnd = (event: TransitionEvent): void => {
     if (event.propertyName === this.activeTransitionProp) {
-      this.open || this.active ? this.emitOpenCloseEvent("open") : this.emitOpenCloseEvent("close");
+      this.open || this.active ? this.onOpen() : this.onClose();
     }
   };
 
+  /* *
```

Comment:
Is this comment needed? I don't see any canceling nor handling of `transitionstart`? If needed, can you remove the extra space and `*` after the first `/*` (nitpick)?

Reply:
> Is this comment needed? I don't see any canceling nor handling of `transitionstart`? If needed, can you remove the extra space and `*` after the first `/*` (nitpick)?

We don't really do anything on transitionstart.
The note is to indicate that if there is a delay it will emit before the start, but if there is none, it'll emit at the start.